app/models/site.rb

```diff
@@ -21,10 +21,15 @@ def reviewed
   end
 
   def self.search(query)
-    where(
-      "plainto_tsquery(?) @@ " +
-      "to_tsvector('english', name || ' ' || description)",
-      query
-    )
+    if query
+      where(
+        "plainto_tsquery(?) @@ " +
+        "to_tsvector('english', LOWER(name) || ' ' || description)",
+        query
+      )
+      where(['LOWER(name) LIKE ?',"%" + query.downcase + "%"])
```

Comment:
Space missing after comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.